third_party/silabs/silabs_board.gni

```diff
@@ -72,6 +75,10 @@ if (silabs_board == "") {
 
 assert(silabs_board != "", "silabs_board must be specified")
 
+if (silabs_board == "BRD2605A") {
+  sl_enable_rgb_led = true
+}
```

Comment:
This should go in the if-else for the BRD2605 that is at line 82.

Reply:
done @mkardous-silabs